mayastor/src/target/iscsi.rs

```diff
@@ -69,9 +70,9 @@ impl RpcErrorCode for Error {
 
 type Result<T, E = Error> = std::result::Result<T, E>;
 
-/// iscsi target port number
-const ISCSI_PORT_NEXUS: u16 = 3260;
-const ISCSI_PORT_REPLICA: u16 = 3262;
+/// Default iSCSI target port numbers
```

Comment:
Now that the port numbers are defined by the config subsystem, we should be able to get rid of this consts, right? That is to say, if you add them to the opts.rs (iirc)

Reply:
Have moved these constants (and also those for the nvmf ports) to opts.rs